src/chocolatey.resources/helpers/functions/Install-ChocolateyPath.tests.ps1

```diff
@@ -0,0 +1,126 @@
+﻿. $PSScriptRoot\Install-ChocolateyPath.ps1
```

Comment:
This file is in a non-good location. If we are going to introduce Pester tests again (or I should say when we do, as they are in the original chocolatey/chocolatey), they need to be in a test project directory.